### PR TITLE
cheeky cheeky test for @Clemcarle

### PR DESCRIPTION
**Pull request by swom**
Helloooo,
I am so excited, my first pull request to you!
This is a cheeky test, that you will need to solve, eheh! Or maybe I will. Let me know if it is all in good order.

### CODE REVIEW

File: simulation.cpp [hunk 1]
```diff
@@ -368,16 +368,25 @@ void test_simulation() noexcept//!OCLINT test may be many
...
 
-//#define FIX_ISSUE_30
-#ifdef FIX_ISSUE_30
+//#define FIX_ISSUE_68
+#ifdef FIX_ISSUE_68
+    ///Ex issue #30 test
+    ///#define FIX_ISSUE_30
+    ///#ifdef FIX_ISSUE_30
+
     ///individuals in a pop are selected based on how closely they match the current_env_target_value
```
Comment (Clemcarle) [inline simulation.cpp:377]: Change comment?

File: simulation.cpp [hunk 2]
```diff
@@ -386,8 +395,8 @@ void test_simulation() noexcept//!OCLINT test may be many
         change_nth_ind_net(s, worst_ind, worst_net);
         auto resp_worst = response(get_nth_ind(s, worst_ind))[0];
 
-        assert(resp_best == get_current_env_value(s));
-        assert(resp_worst != get_current_env_value(s));
+        assert(resp_best == optima_is_response_of_grn_first_ind(s.get_pop()));
```
Comment (Clemcarle) [inline simulation.cpp:398]: The environmental function are set up to work with inputs, I don't think feeding them a population is going to work.

File: simulation.cpp [hunk 3]
```diff
@@ -386,8 +395,8 @@ void test_simulation() noexcept//!OCLINT test may be many
...
+        assert(resp_worst != optima_is_response_of_grn_first_ind(s.get_pop()));
```
Comment (Clemcarle) [inline simulation.cpp:399]: same here, should use inputs instead of pop

**Review by Clemcarle — CHANGES_REQUESTED** [review]
See comments, also the comments in the code are outdated and could be updated.

File: simulation.cpp (hunk 2 above)
Comment (swom) [reply simulation.cpp:398]: Well spotted, 
That is exactly what I thought was the problem when you asked me to change something. I will try to think of a workaround.

File: simulation.cpp (hunk 3 above)
Comment (swom) [reply simulation.cpp:399]: 👍

**Review by Clemcarle — APPROVED** [review]
Ok, the comments still need to be changed but I think this approach works

**Pull request merged**